### PR TITLE
[Feature] View poster image on fullscreen, when its clicked or tapped.

### PR DESCRIPTION
**Pull request by Jacekun**
ready for merge.

### CODE REVIEW

**Comment by LagradOst** [comment]
No, dont do it like this. Make a dialog that you show

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt [hunk 1]
```diff
@@ -1358,9 +1360,29 @@ class ResultFragment : Fragment(), PanelsChildGestureRegionObserver.GestureRegio
...
-                            result_poster?.setImage(d.posterUrl)
-                            result_poster_blur?.setImageBlur(d.posterUrl, 10, 3)
+                        val posterImageLink = d.posterUrl
+                        if (!posterImageLink.isNullOrEmpty()) {
+                            result_poster?.setImage(posterImageLink)
+                            result_poster_blur?.setImageBlur(posterImageLink, 10, 3)
+                            //Full screen view of Poster image
+                            result_poster_holder?.setOnClickListener {
+                                try {
+                                    if (context != null) {
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt:1370]: do context?.let { ctx ->

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt [hunk 2]
```diff
@@ -1358,9 +1360,29 @@ class ResultFragment : Fragment(), PanelsChildGestureRegionObserver.GestureRegio
...
+                                        //TODO: UI setting for either 960x1280 or 960x1600
+                                        val bitmap = result_poster.drawable.toBitmap(960, 1600)
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt:1372]: Is there any way of doing this in a dynamic way?

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt (hunk 1 above)
Comment (Jacekun) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt:1370]: will change it

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt (hunk 2 above)
Comment (Jacekun) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragment.kt:1372]: I've tried fill and wrap content but it shows the image from its original size. scaleType fitXY gives a wacky display. I'll need to fiddle with it some more to see but for now, it works hardcoded

**Pull request merged**